### PR TITLE
New info table

### LINKED ISSUE #103: Saving the date of the scrape
# Why?
As we intend to start checking if the current data of TTS is up to date, a new table must be created.
For now, this table will store the date when the scrapper was run. In the future it can save other useful information.

### PR DESCRIPTION
**Pull request by diogotvf7**
# What was done?
A new table was created: 
```sql
CREATE TABLE `info` (
  `date` DATETIME PRIMARY KEY
);
```
For now, this table stores the date when the scrapper was run, in the future it might store more relevant information.

Closes #103

### CODE REVIEW

File: src/Makefile [hunk 1]
```diff
@@ -24,6 +25,11 @@ schedule_professor:
 professors:
 	scrapy crawl professors
 
+info:
+	@echo $(shell pwd)
+	@echo $(shell ls)
```
Comment (LuisDuarte1) [inline src/Makefile:30]: Is this needed? or it was a left over for debugging?

**Review by LuisDuarte1 — CHANGES_REQUESTED** [review]
Good work :rocket: , just have some concerns below.

File: src/Makefile (hunk 1 above)
Comment (diogotvf7) [reply src/Makefile:30]: Oopsie 😆

**Review by LuisDuarte1 — APPROVED** [review]
LGTM

**Pull request merged**